src/git.spec.js

```diff
@@ -0,0 +1,11 @@
+import test from 'ava';
+import git from './git-commands';
+
+test('extract git versions return in expected format [, major, minor]', t => {
+  let [, major, minor] = git.versionArray('git version 2.10.1.windows.1');
+  t.is(major + '.' + minor, '2.10');
+  [, major, minor] = git.versionArray('git version 2.15.0');

```

Comment:
Nice variety of test cases. For readability, what do you think about newlines in between the test cases?